portal-bridge/src/bridge.rs

```diff
@@ -76,6 +76,10 @@ impl Bridge {
             BridgeMode::Latest => self.launch_latest().await,
             _ => self.launch_backfill().await,
         }
+        // let test finish uTP transfer before shutting down gossip nodes
```

Comment:
I'm curious, but wouldn't this be a problem for all bridge mode branches? 

Anyway, I do think the need for this timeout goes away after #1010 since the bridge is updated to use the `portal_*TraceGossip` endpoint, which implicitly waits for all utp transfers to finish before responding. If you agree, then I'd advocate for removing this `sleep` since it would be unnecessary

Reply:
I guess it would be I was just thinking from the view point of getting my test working.

Will the bridge not always use portal_*TraceGossip?

I will remove this sleep if that is the case then

Reply:
Yeah, makes sense. As of #1010, the bridge will always use `portal_*TraceGossip` for both beacon & history content. Idk if it'll be like this forever, but I can't think of a good enough reason to suggest it might change, especially since consuming the gossip trace is fairly important for the bridge metrics.

Reply:
Ok I just removed the sleep